cpp/ql/test/library-tests/unnamed/elements.ql

```diff
@@ -3,5 +3,6 @@ import cpp
 from Element e, string s
 where
   not e instanceof Folder and
+  exists(e.toString()) and // Work around `VariableDeclarationEntry.toString()` not holding
```

Comment:
The extractor seems to emit certain global variables with `<unnamed>` as the name of the variable, but with an empty string as the name column in `var_decls`. Empty strings cause `VariableDeclarationEntry.getName()` to not hold, which in turn causes `VariableDeclarationEntry.toString()` to not hold. Not sure where to fix this behavior, either.